<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7108" started-at="2021-05-27T16:10:13 IST" finished-at="2021-05-27T16:10:21 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7108" started-at="2021-05-27T16:10:13 IST" finished-at="2021-05-27T16:10:21 IST">
      <class name="log4jPOM.LoginPageTest">
        <test-method status="PASS" signature="getBrowser()[pri:0, instance:log4jPOM.LoginPageTest@51e2adc7]" name="getBrowser" is-config="true" duration-ms="3627" started-at="2021-05-27T16:10:10 IST" finished-at="2021-05-27T16:10:13 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getBrowser -->
        <test-method status="PASS" signature="getUrl()[pri:0, instance:log4jPOM.LoginPageTest@51e2adc7]" name="getUrl" is-config="true" duration-ms="7008" started-at="2021-05-27T16:10:13 IST" finished-at="2021-05-27T16:10:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUrl -->
        <test-method status="FAIL" signature="registerData1()[pri:0, instance:log4jPOM.LoginPageTest@51e2adc7]" name="registerData1" duration-ms="13" started-at="2021-05-27T16:10:20 IST" finished-at="2021-05-27T16:10:21 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at log4jPOM.LoginPageWithPage.loginCheck(LoginPageWithPage.java)
at log4jPOM.LoginPageTest.registerData1(LoginPageTest.java)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerData1 -->
      </class> <!-- log4jPOM.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
